app/views/docs/functions.phtml

```diff
@@ -503,14 +503,64 @@ static RuntimeResponse &main(const RuntimeRequest &req, RuntimeResponse &res) {
 
 <p><a href="/docs/server/functions#functionsCreateDeployment" rel="noopener"><i class="icon-angle-circled-right margin-start-negative-tiny margin-end-tiny"></i> You can also create new code deployments using the Appwrite server API</a></p>
 
+<h3>Manual Deployment</h3>
+<p>You can also upload your functions to be deployed using the Appwrite console. The example below shows a typical Node.JS function, but the same idea applies to any other language.</p>
+
+<div class="ide margin-bottom">
+    <pre class="line-numbers"><code class="prism" data-prism>.
+├── package.json
+└── src
+    └── main.js
```

Comment:
We could simplify even more. No need for `src` folder. Only thing that would change is entrypoint from `src/main.js` to `main.js`.